webknossos/webknossos/dataset/defaults.py

```diff
@@ -1,13 +1,21 @@
+import os
+
 from ..geometry import Vec3Int
 from .data_format import DataFormat
 
 DEFAULT_WKW_FILE_LEN = 32
 DEFAULT_CHUNK_SHAPE = Vec3Int.full(32)
-DEFAULT_CHUNKS_PER_SHARD = Vec3Int.full(32)
+DEFAULT_DATA_FORMAT = (
+    DataFormat.WKW
+    if not os.environ.get("WK_DEFAULT_DATA_FORMAT") == "zarr3"
```

Comment:
Could you parse the string directly into the `DataFormat` enum? Probably just `DataFormat(os.environ["WK_DEFAULT_DATA_FORMAT"]) if "WK_DEFAULT_DATA_FORMAT" in os.environ else DataFormat.WKW`